packages/gatsby-graphiql-explorer/src/app/app.js

```diff
@@ -4,14 +4,14 @@ import ReactDOM from "react-dom"
 import GraphiQL from "graphiql"
 import GraphiQLExplorer from "graphiql-explorer"
 import { getIntrospectionQuery, buildClientSchema, parse } from "graphql"
-import CodeExporter from "graphiql-code-exporter"
-import snippets from "./snippets"
+// import CodeExporter from "graphiql-code-exporter"
```

Comment:
`graphiql-code-exporter` wants to use `regeneratorRuntime` which previously was included in `graphiql` (we currently use `graphiql@0.17.5`. Graphiql currently is on `1.2.2` and I think it dropped adding `regeneratorRuntime` in `1.0.0` and I didn't want to spend time yet on making it work, so for now this is commented out.

We might want to add this ourselves (and actually transpile `graphiql`) if we want to support IE11 for graphiql (it should work now, but it will likely stop working once we upgrade from currently used graphiql version)

Reply:
oh I think gatsby users would have a lot of reason to want features in the newer versions, I can help get this all wrangled i think!